store/sqlstore/store.go

```diff
@@ -1284,3 +1294,12 @@ func (ss *SqlStore) GetAppliedMigrations() ([]model.AppliedMigration, error) {
 
 	return migrations, nil
 }
+
+func (ss *SqlStore) Explain(query string, args []any) (string, error) {
+	var explain []string
+	if err := ss.GetMasterX().Select(&explain, "EXPLAIN "+query, args...); err != nil {
```

Comment:
This has potential for SQL injections. cc @jupenur

Reply:
Not only potential, it is almost for sure a SQL injection. The idea is, this shouldn't be enabled in production ever. Anyway, just in case, I going to try to make it no vulnerable.

Reply:
Actually, I just realized, for postgres there is no way to do a second query as part of the same `Select` there, so at least in postgres we are save of multiple queries injection there. Said that, you can still do something like `ANALYZE whatever query`, that would end up being `EXPLAIN ANALYZE` what leads to code execution. I changed it to avoid queries starting with `ANALYZE`.

Reply:
I think this should be done